tests/rbe_repo/BUILD

```diff
@@ -867,16 +870,31 @@ sh_test(
 
 # Verifies that all containers listed in the versions.bzl file used by
 # rbe_autoconfig are available for pulling.
-# This test requires docker to be present on the path.
-sh_test(
+rbe_container_test(
     name = "rbe_container_test",
     size = "large",
```

Comment:
nit: can this test now be classified as "small"?

Reply:
This only affects the timeout so can be left as is. "large" sets it to 30mins.